pkg/middleware/logger.go

```diff
@@ -33,11 +36,11 @@ func Logger() macaron.Handler {
 		rw := res.(macaron.ResponseWriter)
 		c.Next()
 
-		timeTakenMs := time.Since(start) / time.Millisecond
+		timeTaken := time.Since(start) / time.Millisecond
```

Comment:
Dont think I changed those but looks fine :)

Reply:
You didn't! this change was introduced in 3d3a7cbba80e02017455947ec7608a03a8c95e44

Reply:
I wonder how come it got included in this PR, but doesn't make any difference.